pkg/catalog/manager/traverse.go

```diff
@@ -38,25 +41,12 @@ func (m *Manager) traverseAndUpdate(repoPath, commit string, cmt *CatalogInfo, t
 		templateNamespace = projectCatalog.Namespace
 	}
 
-	templateMap, err := m.getTemplateMap(catalog.Name, templateNamespace)
-	if err != nil {
-		return err
-	}
-	// list all templateContent tag
-	templateContentList, err := m.templateContentLister.List("", labels.NewSelector())
-	if err != nil {
-		return err
-	}
-	templateContentMap := map[string]struct{}{}
-	for _, t := range templateContentList {
-		templateContentMap[t.Name] = struct{}{}
-	}
-
 	newHelmVersionCommits := map[string]v3.VersionCommits{}
 	var errs []error
 	var terrors []error
 	createdTemplates, updatedTemplates, deletedTemplates := 0, 0, 0
 	for chart, metadata := range index.IndexFile.Entries {
+		logrus.Infof("Catalog [%s] found chart template for [%s]", catalog.Name, chart)
```

Comment:
maybe move to debug?

Reply:
Fixed and associated logging for this PR cleaned up